src/components/core/geoChart/geoChartDataSource.ts

```diff
@@ -104,6 +127,86 @@ function transformPushpinDataSource(
     return features;
 }
 
+function getDrillableProps(dataSourceProps: IGeoDataSourceProps, dataIndex: number): IDrillableProps {
+    const { drillableItems, drillConfig, execution, geoData } = dataSourceProps;
+    const { executionResponse } = execution;
+    const { dimensions } = executionResponse;
+
+    const measureGroup = findMeasureGroupInDimensions(dimensions);
+    const measureHeaders: Execution.IMeasureHeaderItem[] = get(measureGroup, "items", []).slice(0, 2);
```

Comment:
with one single level, we should avoid using `get` to not break Ts

Reply:
updated